controllers/core/openstackversion_controller.go

```diff
@@ -240,10 +238,22 @@ func (r *OpenStackVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			corev1beta1.OpenStackVersionMinorUpdateOVNControlplane,
 			corev1beta1.OpenStackVersionMinorUpdateReadyMessage)
 
+		// minor update for Dataplane OVN
+		if !openstack.DataplaneNodesetsOVNControllerDeployedCheck(instance, dataplaneNodesets) {
```

Comment:
The current logic in DataplaneNodesetsOVNControllerDeployedCheck will return false if there are NodeSets that are not ready even if that nodeset does not need to deploy OVNController. So I think we need to change the logic there to ignore these nodeset